x-pack/test/security_solution_cypress/cypress/cypress_serverless.config.ts

```diff
@@ -22,10 +22,12 @@ export default defineCypressConfig({
   viewportWidth: 1680,
   numTestsKeptInMemory: 10,
   env: {
+    IS_SERVERLESS: true,
```

Comment:
This value is already set in our test code, more specifically in the parallel script. Having it here as well may lead to misunderstandings or wrong values, we must have it in just one place. Personally, I would prefer to have it outside this configuration file.